docs/src/man/basics.md

```diff
@@ -0,0 +1,2170 @@
+                             DataFrame)
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67  male        2  own      NA               little          ⋯
+    2 │     1     22  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> german = copy(german_ref); # It will copy the data frame
+```
+
+Now let's talk about the given code block:
+```julia
+german_ref = CSV.read(joinpath(dirname(pathof(DataFrames)),
+                               "..", "docs", "src", "assets", "german.csv"),
+                      DataFrame)
+```
+- we are storing `german.csv` file in the DataFrames.jl repository to make user's life easier and
+  avoid having to download it each time;
+- `pathof(DataFrames)` gives us the full path of the file that was used to import the DataFrames.jl package;
+- first we split the directory part from it using `dirname`;
+- then from this directory we need to move to the directory where `german.csv` is stored; we use
+  `joinpath` as this is a recommended way to compose paths to resources stored on disk in an operating
+  system independent way (remember that Widnows and Unix differ as they use either `/` or `\` as path
+  separator; the `joinpath` function ensures we are not running into issues with this);
+- then we read the CSV file; the second argument to `CSV.read` is `DataFrame` to indicate that we want to
+  read in the file into a `DataFrame` (as `CSV.read` allows for many different target formats of data it
+  can read-into).
+
+You can see that DataFrames.jl (unlike Python's Pandas) displays the data type of the column,
+In our case, it is an `Int64`, or `String`.
```

Comment:
this section was about constructors. I can see that you have shown only a kwarg constructor. Why?

Reply:
Working on adding more. I thought it will be enough but I was wrong. :(

Reply:
If you think it is enough then I am OK, but just mention there are other constructors and then probably make a separate section about reading in files to a `DataFrame`, as now you have squeezed it into a section on constructors.